tools/compogen/pkg/gen/resources/templates/readme.mdx.tmpl

```diff
@@ -60,12 +71,48 @@ ${connection.<my-connection-id>}`.
 | {{ .Title }}{{ if .Required }} (required){{ end }} | `{{ .ID }}` | {{ .Type }} | {{ .Description }} |{{ end }}
 {{ end }}
 
+{{ if $task.InputObjects }}
+{{ range $idx, $objectSchemaMap := $task.InputObjects }}
+{{ range $key, $schema := $objectSchemaMap }}
+{{ if $schema.Properties }}
+#### {{ $key }}
+
+| Field | Field ID | Type | Note |
+| :--- | :--- | :--- | :--- |{{ range $propKey, $prop := $schema.Properties }}
+| {{ $prop.Title }} | {{ $propKey }} | {{ $prop.Type }} | {{ $prop.Description }} {{ if .Enum }} There are options: <br/>- {{ enumValues .Enum }}  {{ end }} |{{ end }}
```

Comment:
```suggestion
| {{ $prop.Title }} | `{{ $propKey }}` | {{ $prop.Type }} | {{ $prop.Description }} {{ if .Enum }} There are options: <br/>- {{ enumValues .Enum }}  {{ end }} |{{ end }}
```

Reply:
When the nested object isn't a `oneOf` I'd rather have the sub objects in the same table, e.g. 

```md
| Token Count | `text-chunks[].token-count` | integer | Count of tokens in a chunk |
```

Reply:
I will do object section first to check the result!
If it is not so good, I will try this solution!